scout/server/config.py

```diff
@@ -99,6 +98,19 @@
 #            },
 #        ],
 #    },
+#    {
+#        "name": "local_tracks",
+#        "access": "public",
+#        "tracks": [
+#            {
+#                "name": "NCBI RefSeq & Ensembl transcripts from MANE",
```

Comment:
This name is super long, perhaps we could call the track something shorter

Reply:
Yes, "MANE" comes to mind.. 😸